### ISSUE TITLE
TextField Cut/Copy/Paste popup menu is clipped on iOS using Larger Text setting

### ISSUE THREAD

**Issue by d4manek** [flutter/flutter#47365]
## Steps to Reproduce
1. Run the Flutter Gallery sample application on a real iOS device.
2. Select Material > Text fields.
3. Type some text into one of the fields.
4. Double-tap on the text just entered to expose the Cut | Copy | Paste popup menu.
5. Notice the popup menu text is clipped on the bottom half.

## Flutter doctor
```console
[✓] Flutter (Channel beta, v1.12.13+hotfix.6, on Mac OS X 10.15.1 19B88, locale en-US)
    • Flutter version 1.12.13+hotfix.6 at /Users/name/Development/flutter
    • Framework revision 18cd7a3601 (7 days ago), 2019-12-11 06:35:39 -0800
    • Engine revision 2994f7e1e6
    • Dart version 2.7.0

 
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at /Users/name/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-28, build-tools 28.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.1, Build version 11A1027
    • CocoaPods version 1.8.4

[✓] Android Studio (version 3.5)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 42.1.1
    • Dart plugin version 191.8593
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)

[✓] VS Code (version 1.39.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.5.1

[✓] Connected device (1 available)
    • iPhone 11 Pro • E8363CE9-9E6B-4344-9096-654BEA2FE787 • ios • com.apple.CoreSimulator.SimRuntime.iOS-13-1 (simulator)

• No issues found!
```

## Screenshot
![IMG_F76542EBBF82-1](https://user-images.githubusercontent.com/17515596/71114693-9741d200-218d-11ea-9fd8-d3963a163735.jpeg)

**Comment by d4manek** [comment]
Any updates on this? It's hard to ship an application with a such a glaring cosmetic issue like this one. Thank you.

**Comment by eugene-kalaganov** [comment]
I sounds weird but I faced the same issue and noticed it happened after I added paddings in buttonTheme
![Screenshot 2020-03-03 at 15 17 21](https://user-images.githubusercontent.com/15782257/75779513-6d4a8580-5d62-11ea-9c89-e2788ae990a8.png)

**Comment by d4manek** [comment]
It happens all the time for me, even without customizing the buttonTheme. Plus it happens in the stock Flutter Gallery app with no modifications.

**Comment by eugene-kalaganov** [comment]
just tried with Gallery app 
with master and beta channels, it works well

**Comment by d4manek** [comment]
You are correct. It happens with the largest Text Size setting in Settings > Display & Brightness. It gets worse if you choose even larger text in Settings > Accessibility > Display & Text Size. Seems like the Cut | Copy | Paste popup should be coded to accommodate varying system font settings instead of just clipping it. The native popup in iOS will actually stack the commands vertically when the font gets too large, which is a pretty good approach.

**Comment by gmaggio** [comment]
This is not good. Surely developers need to make custom buttons via the theme. The **Cut/Copy/Paste** popup menu shouldn't be using the same theme. Can't flutter just use the platform's built-in **cut/copy/paste** menu?

**Comment by markusaksli-nc** [comment]
Reproducible on the latest master `1.22.0-10.0.pre.221`.
<details>
 <summary>flutter doctor -v</summary>

```
[✓] Flutter (Channel master, 1.22.0-10.0.pre.221, on Mac OS X 10.15.6 19G2021 x86_64, locale en-GB)
    • Flutter version 1.22.0-10.0.pre.221 at /Users/markus/development/flutter_master
    • Framework revision fa646339d6 (5 hours ago), 2020-09-16 17:47:04 -0700
    • Engine revision 933f811d94
    • Dart version 2.10.0 (build 2.10.0-135.0.dev)

 
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.1)
    • Android SDK at /Users/markus/Library/Android/sdk
    • Platform android-30, build-tools 30.0.1
    • Java binary at: /Users/markus/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-0/193.6626763/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.7)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.7, Build version 11E801a
    • CocoaPods version 1.9.3

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.0)
    • Android Studio at /Users/markus/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-0/193.6626763/Android Studio.app/Contents
    • Flutter plugin version 48.1.2
    • Dart plugin version 193.7361
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)

 
[✓] Connected device (5 available)            
    • Nevercode’s iPhone (mobile)         • b668e524315069f3db3661ac11ff1f66afafebdb • ios            • iOS 13.7
    • iPhone SE (2nd generation) (mobile) • EE7A6A68-952D-4827-A65C-743575E18C90     • ios            • com.apple.CoreSimulator.SimRuntime.iOS-13-7 (simulator)
    • macOS (desktop)                     • macos                                    • darwin-x64     • Mac OS X 10.15.6 19G2021 x86_64
    • Web Server (web)                    • web-server                               • web-javascript • Flutter Tools
    • Chrome (web)                        • chrome                                   • web-javascript • Google Chrome 85.0.4183.102

• No issues found!
```
</details>

Setting on iOS 13.7

<img src="https://user-images.githubusercontent.com/68219924/93426455-be683400-f8c4-11ea-85cf-dd619716c793.jpeg" height="400" />

Native vs Flutter

<img src="https://user-images.githubusercontent.com/68219924/93426450-bd370700-f8c4-11ea-907c-6a0c2c8a433f.jpeg" height="400" /><img src="https://user-images.githubusercontent.com/68219924/93426456-be683400-f8c4-11ea-9f64-fdb1e986fa96.jpeg" height="400" />

**Comment by pdsliuzhen** [comment]
Hello,guys, Has the problem been solved?

**Comment by pdsliuzhen** [comment]
Hello,guys, Has the problem been solved?@d4manek

**Comment by maheshmnj** [comment]
Reproduces on latest master 3.7 and stable 3.3

<img width="366" alt="image" src="https://user-images.githubusercontent.com/31410839/207973490-8a492fbe-b5de-49e0-93d3-35f972fbba1a.png">


https://user-images.githubusercontent.com/31410839/207973418-59de2214-45a0-4aa9-8281-68acd353fc0a.mov

<details>
<summary>flutter doctor -v</summary>

```dart
import 'package:flutter/material.dart';

void main() => runApp(MyApp());

class MyApp extends StatelessWidget {
  const MyApp({super.key});

  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Sample',
      home: Scaffold(
        appBar: AppBar(
          title: const Text('Sample'),
        ),
        body: const Center(
          child: TextField(),
        ),
      ),
    );
  }
}
```
</details> 


<details>
<summary>flutter doctor -v (mac)</summary>

```
[✓] Flutter (Channel master, 3.7.0-3.0.pre.33, on macOS 12.6 21G115 darwin-arm64, locale en-IN)
    • Flutter version 3.7.0-3.0.pre.33 on channel master 
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 5201856805 (38 minutes ago), 2022-12-05 18:27:21 -0800
    • Engine revision a309d239c4
    • Dart version 2.19.0 (build 2.19.0-463.0.dev)
    • DevTools version 2.20.0
    • If those were intentional, you can disregard the above warnings; however it is recommended to use "git" directly
      to perform update checks and upgrades.

[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.0-rc4)
    • Android SDK at /Users/mahesh/Library/Android/sdk
    • Platform android-33, build-tools 33.0.0-rc4
    • ANDROID_HOME = /Users/mahesh/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.12+0-b1504.28-7817840)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 14.0.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 14A400
    • CocoaPods version 1.11.3

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2021.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.12+0-b1504.28-7817840)

[✓] IntelliJ IDEA Community Edition (version 2021.2.1)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    • Flutter plugin version 61.2.4
    • Dart plugin version 212.5080.8

[✓] VS Code (version 1.70.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.55.20221129

[✓] Connected device (3 available)
    • iPhone 12 Pro (mobile) • 026D5789-9E78-4AD5-B1B2-3F8D4E7F65E4 • ios            •
      com.apple.CoreSimulator.SimRuntime.iOS-14-5 (simulator)
    • macOS (desktop)        • macos                                • darwin-arm64   • macOS 12.6 21G115 darwin-arm64
    • Chrome (web)           • chrome                               • web-javascript • Google Chrome 108.0.5359.94

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```
```
[✓] Flutter (Channel stable, 3.3.9, on macOS 12.6 21G115 darwin-arm, locale en-IN)
    • Flutter version 3.3.9 on channel stable at /Users/mahesh/Development/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision b8f7f1f986 (24 hours ago), 2022-11-23 06:43:51 +0900
    • Engine revision 8f2221fbef
    • Dart version 2.18.5
    • DevTools version 2.15.0

[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.0-rc4)
    • Android SDK at /Users/mahesh/Library/Android/sdk
    • Platform android-33, build-tools 33.0.0-rc4
    • ANDROID_HOME = /Users/mahesh/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.12+0-b1504.28-7817840)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 14.0.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 14A400
    • CocoaPods version 1.11.3

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2021.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.12+0-b1504.28-7817840)

[✓] IntelliJ IDEA Community Edition (version 2021.2.1)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    • Flutter plugin version 61.2.4
    • Dart plugin version 212.5080.8

[✓] VS Code (version 1.70.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.53.20221101

[✓] Connected device (3 available)
    • iPhone 12 Pro (mobile) • 026D5789-9E78-4AD5-B1B2-3F8D4E7F65E4 • ios            •
      com.apple.CoreSimulator.SimRuntime.iOS-14-5 (simulator)
    • macOS (desktop)        • macos                                • darwin-arm64   • macOS 12.6 21G115 darwin-arm
    • Chrome (web)           • chrome                               • web-javascript • Google Chrome 107.0.5304.110

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```
</details>

**Comment by Ignacio1110** [comment]
this issue is related to #136527 
already solved.

**Comment by justinmc** [comment]
Thank you @Ignacio1110, I've verified with the code in https://github.com/flutter/flutter/issues/47365#issuecomment-693944834 and this is indeed fixed.

<img width="363" alt="Image" src="https://github.com/user-attachments/assets/f97b6ee3-dc3f-4902-b685-48b6781439e3" />
